src/database/industry_seestar_helper.erl

```diff
@@ -31,16 +32,27 @@ prepare_insert(NameSpace, Table, Schema, Values) ->
 	    end || {Attribute, AttrType} <- Attributes],
     {lists:flatten(Query), Row}.
 
+-spec prepare_select(iolist(), atom(), [term()], iolist()) -> string().
+prepare_select(NameSpace, Table, Schema, Id) ->
```

Comment:
I kept the `prepare_select/4` to not cause any breaking API changes.

Added DRYness as you recommended.